homeassistant/components/esphome/strings.json

```diff
@@ -1,6 +1,7 @@
 {
   "config": {
     "abort": {
+      "unknown": "[%key:common::config_flow::error::unknown%]",
```

Comment:
We know what the error is, the payload is empty so we should give a better error message here

Reply:
added a new message `mqtt_missing_payload`